packages/quantic/force-app/main/default/lwc/caseClassification/caseClassification.js

```diff
@@ -0,0 +1,195 @@
+import { LightningElement, api } from 'lwc';
+import caseClassificationTitle from '@salesforce/label/c.quantic_CaseClassificationTitle';
+import selectOption from '@salesforce/label/c.quantic_SelectOption';
+import selectAnotherFeature from '@salesforce/label/c.quantic_SelectAnotherFeature';
+import selectProduct from '@salesforce/label/c.quantic_SelectProduct';
+
+/**
+ * A section for a user to classify his case aided by suggestions provided by Coveo Case Assist. There is also a dropdown available to see all available values for a given category.
+ */
+export default class CaseClassification extends LightningElement {
+  labels = {
+    caseClassificationTitle,
+    selectAnotherFeature,
+    selectOption,
+    selectProduct
+  };
+
+  /**
+   * All the options proposed to the user to choose from.
+   * @type {Array}
+   * @defaultValue `[]`
+   */
+  @api options = [];
+
+  /**
+   * The number of suggestions to be shown.
+   * @type {number}
+   * @defaultValue `3`
+   */
+  @api numberOfSuggestions = 3;
```

Comment:
It would be nice to validate that `numberOfSuggestions` is greater or equal to zero. When the value is invalid we could either raise an error, or log a warning and continue with `numberOfSuggestions = 0`. I think both approaches could work.